tests.py

```diff
@@ -72,6 +72,17 @@ def test_replace_urls(self):
         self.assertTrue(os.path.isfile(os.path.join(dir_path, "testfiles", "testurls_output.md")))
         self.assertTrue(os.path.isfile(os.path.join(dir_path, "testfiles", "txturls_output.txt")))
 
+        # Clean test folder of results so tests can be repeated.
```

Comment:
Hi, thanks! This looks good. What do you think of automating the detection process? For instance, we could have:

```python
import glob
created_output_files = glob.glob(os.path.join(dir_path, "testfiles/*_output.*"))

```

Then proceed with 

```python
for file in created_output_files:
......
```

I would sugggest renaming `garbage_test_files` 😂 to something else (like `created_output_files` or `left_over_files`)